docs/content/getting-started/setup-openfga/overview.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Setup OpenFGA
+description: Setting up an OpenFGA server
+sidebar_position: 1
+slug: /getting-started/setup-openfga/overview
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+  CardGrid,
+} from "@components/Docs";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Setup OpenFGA
+Follow the guides below to setup an OpenFGA server.
+
+<DocumentationNotice />
+
+<CardGrid
+  middle={[
+    {
+      title: 'Docker Setup Guide',
+      description: 'How to setup an OpenFGA server with Docker.',
+      to: 'docker',
```

Comment:
```suggestion
      to: 'getting-started/setup-openfga/docker',
```

Reply:
Sorry, it should be 

`setup-openfga/docker`.  See https://raw.githubusercontent.com/openfga/openfga.dev/main/docs/content/modeling/migrating/overview.mdx as example